examples/move/scratch_off/sources/math.move

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module scratch_off::math {
+
+    /// Largest u64 integer
+    const MODULO_FACTOR: u256 = 18446744073709551615;
+
+    public fun bytes_to_u256(random_32b: &vector<u8>): u256 {
```

Comment:
can change this function to `bytes_to_u128` and only use the first 16 bytes (you don't need u256 below)